src/common/transformations/include/transformations/common_optimizations/mark_subgraphs_to_keep_in_mixed_precision.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/graph_rewrite.hpp"
+#include "openvino/pass/pass.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+namespace pass {
+
+class TRANSFORMATIONS_API MarkSugraphsToKeepInMixedPrecision;
+
+// through this nodes precision sensitiveness is propagated
+extern std::shared_ptr<Node> propagate_through_ops;
+
+}  // namespace pass
+}  // namespace ov
+
+/* MarkSugraphsToKeepInMixedPrecision  marks patterns that should be kept in f32 for
+ * mixed precision inference. Such patterns include: L2Normalize, MVN, Division with small eps values
+ */
+class ov::pass::MarkSugraphsToKeepInMixedPrecision : public ov::pass::ModelPass {
```

Comment:
please simplify the name `MarkNormalizationOperations`

Reply:
renamed to `MarkNormalizationOps`